starknet/src/execution_strategies/simple_quorum.cairo

```diff
@@ -67,3 +67,6 @@ mod SimpleQuorumExecutionStrategy {
         votes_for > votes_against
     }
 }
+// TODO: add unit tests for get_proposal_status
```

Comment:
did you plan to add these on this PR?

Reply:
I wasn't planning on but now that I think about it it does make more sense yeah. I'm on it.

Reply:
29519403023dfe7d2c94e1b61103ea97ba961e9e